woocommerce/storefront
# Remove editor dynamic width feature

Closes #1512.

As discussed in #1520, we want to remove the feature to adapt the editor width to the selected template.

### How to test the changes in this Pull Request:
1. Create a page and switch between the 'Full width' template and the 'Default' template.
2. Verify there are no visible changes in the editor, but when previewing the page in the frontend, the correct template is applied.
3. Since this PR includes some changes to the built process, it might be good to test other JS features to ensure there are no regressions. For example, the sticky add to cart bar that can be added from the Customizer (under the WooCommerce > Product Page section) and appears in (not-external) product pages when you scroll down:
![imatge](https://user-images.githubusercontent.com/3616980/97876050-8eada800-1d1b-11eb-8abe-c046eaac3ca1.png)

Affected [flows & features](https://github.com/woocommerce/storefront/wiki/Testing-Storefront:-flows-and-features):

- Block editor shows a realistic preview of the front end styling.

### Changelog

> Removal – Removed the feature to adapt the editor width to the selected template.
